manifests/init.pp

```diff
@@ -1,6 +1,7 @@
 # Default strongSwan class.
 class strongswan(
   $package_name        = $strongswan::params::package,
+  $version             = $strongswan::params::version,
```

Comment:
Maybe `package_version` would be a better name for this parameter?

Reply:
or `package_ensure` as its default value is `installed`.